### ISSUE TITLE
Implement Multivariate linear regression

### ISSUE THREAD

**Issue by balapriyac** [sanscript-tech/hacking-tools-scripts#92]
- Use only NumPy; not scikit-learn
- Use either randomly generated dataset or simple dataset from kaggle
- Pls include EDA section in your notebook for the chosen dataset
- Comment on inferences and metrics for evaluation

**Comment by sujal111** [comment]
Please assign this issue to me. I would love to work on it.

**Comment by tejan-singh** [comment]
please follow contribution guidelines first.

**Comment by sujal111** [comment]
Yes , I am following it. According to that , I am supposed to comment over here so that the maintainer assigns me.

**Comment by tejan-singh** [comment]
You Also have to complete your GitHub profile. At least put your full name to confirm your participation

**Comment by balapriyac** [comment]
I'm assigning it to you @sujal111

**Comment by balapriyac** [comment]
@sujal111 just a small clarification required, could you pls confirm that you've already registered for AOS '20
Because we are unable to find you in the Slack channel assigned for this project

**Comment by sujal111** [comment]
@balapriyac I have joined now.

**Comment by balapriyac** [comment]
Thanks, pls be sure to follow the contribution guidelines going forward
